docs/userguides/managing.md

```diff
@@ -0,0 +1,53 @@
+# Managing Your Platform
```

Comment:
```suggestion
# Managing Workspaces and Clusters via the Silverback Platform
```

Reply:
![image](https://github.com/user-attachments/assets/7b20bac3-100b-4ef6-868c-8adbe0dce96f)

This is too long of a title so I'm going to shorten it to `# Managing Workspaces and Clusters`